src/app/components/LiteSiteSummary/defaultTranslations.ts

```diff
@@ -1,11 +1,14 @@
-import { Services } from '#app/models/types/global';
-
-export const defaultTranslations = {
+const defaultTranslations: {
+  onboardingMessage: string;
+  toMainSite: string;
+  informationPage: string;
+  informationPageLink?: string;
+  dataSaving: string;
```

Comment:
Shall we use the types defined in the translations.ts file here, instead of duplicating?

Reply:
Good shout!